### PR TITLE
nttw7 start

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw7.html.twig [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+{% endimage %}
+</p>
+
+<p>
+<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2560.1680462251616!2d14.415493751272342!3d50.083140521542994!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x470b94ee4e82dc6f%3A0xa6d97f03353c46c!2sKino%20NFA%20Ponrepo!5e0!3m2!1sen!2sus!4v1678811430423!5m2!1sen!2sus" width="600" height="450" style="border:0;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
+</p>
+
+<h2>Travel Grants</h2>
+
+<p>We're currently working on budgeting for travel grants and will announce a travel grant application on this page and at https://twitter.com/nttwconf when they're ready.</p>
```
Comment (JeromeMartinez) [inline src/NoTimeToWaitBundle/Resources/views/Default/nttw7.html.twig]: missing href

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw7.html.twig [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+{% block body %}
+<h1><strong>No Time to Wait!</strong> *</h1>
+
+<h2>The No Time to Wait Conference</h2>
+<p><a href="https://nfa.cz/">Národní filmový archiv</a> shall host the seventh No Time to Wait conference at their theater in Prague</a>. No Time to Wait is a FREE three-day conference focused on <em>open media</em>, <em>open standards</em>, and <em>digital audiovisual preservation</em> managed by <a href="{{ path('homepage') }}">MediaArea.net</a>. No Time to Wait has always been a conference that blends in-person and online participation and this year we shall work to emphasize online participation in response to the social, travel, and financial restrictions related to the pandemic.</p>
+<p>November 8-10, 2023</p>
+
+<h2>Registration</h2>
+<p>If you'd like to participate in No Time to Wait 6, either in-person or online, please complete <a href="https://bit.ly/nttw7signup">our registration form</a>.</p>
```
Comment (JeromeMartinez) [inline src/NoTimeToWaitBundle/Resources/views/Default/nttw7.html.twig]: 6 --> 7 :-p

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw7.html.twig [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+NTTW is the most open, inspiring, and motivating A/V event, with its lack of pretention and truly inclusive spirit. After NTTW4 many of us is to look at their own A/V projects with renewed energy, ideas for solutions and dedication to our wonderful cultural heritage-related work.</i><br/>
+-- Zsuzsanna Zadori, Senior Audio-visual Archivist, Blinken Open Society Archives</p>
+
+<h2>Want to Host No Time to Wait?</h2>
+
+<p>If you're interested being considered as a host for an upcoming No Time to Wait. Please complete <a href="https://bit.ly/nttwhosts">this interest form</a> and a member of our organizing committee will be in touch with you soon.</p>
+
+<h2>Code of Conduct</h2>
+
+<p>No Time to Wait is governed by the <a href="https://mediaarea.net/Conduct">Contributor Covenant version 2.0</a>. All coordinators, contributors, and participants agree to abide by its terms. To report violations, send an email to <a href="mailto:conduct@mediaarea.net?subject=Code%20of%20Conduct">conduct@mediaarea.net</a> or contact a conference coordinator.</p>
```
Comment (JeromeMartinez) [inline src/NoTimeToWaitBundle/Resources/views/Default/nttw7.html.twig]: Contributor Covenant version 2.1

**Review by JeromeMartinez — CHANGES_REQUESTED** [review]

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw7.html.twig [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+<p>We're currently working on budgeting for travel grants and will announce a travel grant application on this page and at https://twitter.com/nttwconf when they're ready.</p>
+
+<h2>Sponsorship</h2>
+
+<p>The organizers of No Time to Wait strive to sustain this conference as a free registration event through efficient and transparent budgeting and via the generosity of our hosts and sponsors. The sponsors of No Time to Wait help make the discussions, presentations, and momentum of the No Time to Wait event possible and enable the event to maintain a high level of accessibility and involvement. Sponsors shall be acknowledged via the conference's website, printed programs, and presence at the conference. Sponsors may indicate that their contribution is allocated to a specific need, such as funding travel stipends, hosting a reception, or supporting live streaming. To discuss or inquire regarding sponsorship opportunities please <a href="{{ path('ma_contact') }}">please contact us</a>.
+
+<p>Thanks to the following organizations for generously supporting No Time to Wait.</p>
+<ul>
+  <li><a href="https://reto.ch/">Reto Kromer</a></li>
```
Comment (JeromeMartinez) [inline src/NoTimeToWaitBundle/Resources/views/Default/nttw7.html.twig]: @retokromer fine for you?

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw7.html.twig (hunk 3 above)
Comment (JeromeMartinez) [reply src/NoTimeToWaitBundle/Resources/views/Default/nttw7.html.twig]: (2.1 because I'll change it to 2.1 very soon)

**Review by JeromeMartinez — APPROVED** [review]

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw7.html.twig (hunk 4 above)
Comment (retokromer) [reply src/NoTimeToWaitBundle/Resources/views/Default/nttw7.html.twig]: You can write `AV Preservation by reto.ch` also this year.

**Review by retokromer — APPROVED** [review]

**Comment by JeromeMartinez** [comment]
It is online at https://mediaarea.net/NoTimeToWait7 a.k.a. https://mediaarea.net/nttw7.

**Pull request merged**